code/datums/components/crafting/recipes.dm

```diff
@@ -92,4 +92,38 @@
 
 	var/obj/item/weapon/material/twohanded/spear/S = result
 	S.set_material(M.material.name)
-	qdel(M)
\ No newline at end of file
+	qdel(M)
+
+/datum/crafting_recipe/shortbow
+	name = "Shortbow"
+	result = /obj/item/weapon/gun/launcher/crossbow/bow
```

Comment:
Nit because I have strong onions about this stuff: [What's a weapon?](https://github.com/PolarisSS13/Polaris/pull/8458)

Reply:
I'm not done with that so this is fine for now - I can always add a finishing commit to address interim changed paths.